### PR TITLE
Refactor OrgForm + small updates

### PR DESCRIPTION
**Pull request by rebecca-caktus**
The first 5 commits contain miscellaneous code quality updates that I ran across.

The last 5 commits refactor `OrgForm`:

* Use Django's defaults wherever possible. I kept the "best" `help_text` between model definitions & form definitions so there's a new migration.
* Separate the login creation functionality since it is only relevant in one place & doesn't need to be a `ModelForm`.
* Add the `logo` field to the `OrgCRUDL.Create` view.
* Same order of fields on `OrgCRUDL.Create` and `OrgCRUDL.Update`.

### CODE REVIEW

File: dash/orgs/views.py [hunk 1]
```diff
@@ -195,13 +141,10 @@ def get_context_data(self, **kwargs):
 
     class Create(SmartCreateView):
         form_class = OrgForm
-        fields = ('name', 'language', 'subdomain', 'domain', 'timezone',
-                  'administrators', 'api_token')
+        exclude = ('is_active',)
```
Comment (norkans7) [inline dash/orgs/views.py]: We use fields explicitly to avoid future changes o the model and forget to update this exclude

**Comment by norkans7** [comment]
Looks good, just revert exclude to fields

**Comment by rebecca-caktus** [comment]
Updated.

**Comment by norkans7** [comment]
Great, thanks

**Pull request merged**